mcp_server/prowler_mcp_server/prowler_hub/server.py

```diff
@@ -0,0 +1,479 @@
+"""
+Prowler Hub MCP module
+
+Provides access to Prowler Hub API for security checks and compliance frameworks.
+"""
+
+from typing import Optional, Any
+import httpx
+from fastmcp import FastMCP
+
+# Initialize FastMCP for Prowler Hub
+hub_mcp_server = FastMCP("prowler-hub")
+
+# API base URL
+BASE_URL = "https://hub.prowler.com/api"
+
+# HTTP client configuration
+client = httpx.Client(
+    base_url=BASE_URL, timeout=30.0, headers={"Accept": "application/json"}
+)
+
+# GitHub raw content base URL for Prowler checks
+GITHUB_RAW_BASE = (
+    "https://raw.githubusercontent.com/prowler-cloud/prowler/refs/heads/master/"
+    "prowler/providers"
+)
+
+# Separate HTTP client for GitHub raw content
+github_client = httpx.Client(
+    timeout=30.0,
+    headers={
+        "Accept": "*/*",
+        "User-Agent": "prowler-mcp-server/1.0",
```

Comment:
```suggestion
        "User-Agent": "prowler-mcp-server/0.1.0",
```

Why not to use the version?